source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/product-definition-multi-select/index.tsx

```diff
@@ -4,10 +4,11 @@ import { withTranslations } from '@lib/i18n';
 import { withNetwork as withProductDefinitions } from '@data/containers/resources/product-definition/list';
 import { withLoader } from '@data/containers/with-loader';
 
-import { Display } from './display';
+//import { Display } from './display';
+import { MultiSelect } from '@ui/components/inputs/multi-select';
 
 export default compose(
   withProductDefinitions(),
   withLoader(({ error, productDefinitions }) => !error && !productDefinitions),
   withTranslations,
-)(Display);
\ No newline at end of file
+)(MultiSelect);
```

Comment:
how can I define what type is the component going to use?

Reply:
ah ok, I understand now -- https://github.com/acdlite/recompose/blob/master/docs/API.md#withprops

Reply:
ok got it